### DISCUSSION TITLE
2023-11-08

### DISCUSSION THREAD

**Discussion by oauch** [10-rolling/rollingPolling#43] [Daily_Meeting]
### ✅ 어제 해낸 일

- [x] 스타일 수정
- [x] Headr 컴포넌트 구현중
- [x] 메세지 보내기 페이지 구현중
- [x] POST 페이지 구현중

### 📝 회의내용

- hook
다 빼자

- page
page에 작업 (만들어놓은 컴포넌트 쓰면서)

- 배포
지금 당장 안해도 된다

- 각자 어디까지 했나용
슬희 : 수정하기, 카드
수님 : 카카오 공유하기, 굉장히 많이함, 우는중
수영님 : background img api 받아와야 돼용
윤혁님 : 많이함

- tmi
슬희 : 16시 pt, 상주하는 강아지 (=벨), 같이 운동을 강아지랑!
수님 : 집순이다.
수영님 : 밥, 제육볶음 ... 맛있겠다
윤혁님 : 등운동하는 날, 아침에 맨날 감

수영님 할아버지님이 수님의 가족을 아신다 ... 호덜덜
현재 회장은 아니심 .. (춘천 회장)
현재는 이사님!
같은 곳에서 묻어진다 .... 😄
수영님은 물에서 묻어지겠다. **수영**이기 때문 ㅋㅋ

### 💪🏻 오늘 할일

- 각자 열심히 개발하자

### 🤔 특이사항

- 어제 헬스장 끝나고 나오다가 누가 저보고 식겁하셨어요

**Comment by hanseulhee** [comment]
흠 좀 늦게 올라왔네요 ....

**Comment by oauch** [comment]
죄송합니다

**Reply by hanseulhee** [reply]
괜찮아요 하하 넓은 아량으로
